beets/ui/commands.py

```diff
@@ -92,6 +97,14 @@ def _print_rows(names):
     print_("Album fields:")
     _print_rows(library.Album.all_keys())
 
+    with lib.transaction() as tx:
+        print_("Item attributes:")
```

Comment:
Actually, let's change the terminology here and add the word "flexible" for clarity.